ui/cap-react/src/components/partials/Menu.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect, useRef } from "react";
+import PropTypes from "prop-types";
+import Box from "grommet/components/Box";
+import ReactDOM from "react-dom";
+
+import { AiOutlineUser } from "react-icons/ai";
+
+const Menu = ({
+  icon = <AiOutlineUser size={23} />,
+  children,
+  top = 48,
+  right = 3,
+  bottom = null,
+  left = null,
+  background = ""
+}) => {
+  const [expanded, setExpanded] = useState(false);
+  const dropDownMenu = useRef(null);
+
+  useEffect(() => {
+    document.addEventListener("keydown", escFunction, false);
+    return () => {
+      document.removeEventListener("keydown", escFunction, false);
+    };
+  }, []);
+
+  const escFunction = event => {
+    // when the user clicks the ESC button
+    // the menu will hide
+    if (event.keyCode === 27) {
+      setExpanded(false);
+    }
+  };
+
+  // useEffect in order to capture clicks that are not in the menu
+  useEffect(
+    () => {
+      function handleClickOutside(event) {
+        const dom = ReactDOM.findDOMNode(dropDownMenu.current);
+        if (dropDownMenu.current && !dom.contains(event.target)) {
+          setExpanded(false);
+        }
+      }
+      // Bind the event listener
+      document.addEventListener("mousedown", handleClickOutside);
+      return () => {
+        // Unbind the event listener on clean up
+        document.removeEventListener("mousedown", handleClickOutside);
+      };
+    },
+    [dropDownMenu]
+  );
+
+  return (
+    <Box ref={dropDownMenu}>
+      <Box onClick={() => setExpanded(!expanded)}>{icon}</Box>
+      <Box
+        style={{
+          position: "absolute",
+          top: `${top}px`,
+          right: `${right}px`,
+          left: `${left}px`,
+          bottom: `${bottom}px`,
+          zIndex: 1,
```

Comment:
needs to be higher e.g. `9999`. Check draftItem page for example where half menu is hidden when open

Reply:
Updated to high value so even in the future there would be no problem